helpers.py

```diff
@@ -102,3 +102,152 @@ def downloadCreateDataList(itemList,site,format,dataPath):
 
     except Exception:
         raise Exception(traceback.format_exc())
+
+#============================
+#Amino Percentage based
+# a Sequence and a Letter
+#============================
+def calcAminoPercentage(seq,letter):
+  counter=0
+  for alphabet in seq:
+    if (alphabet==letter):
+      counter=counter+1
+  
+  return ((counter/len(seq))*100)
+
+#============================
+#Rounds Floating Points Number
+#============================
+def roundFloatingpoint(num):
+  return (np.round(num*1000)/1000)
```

Comment:
Why not just round to the 3rd decimal? https://www.w3schools.com/python/ref_func_round.asp
Also, you shouldn't require numpy for just that.

Reply:
I've included Numpy for some other functionality(in classification part). But i used it for this problem as well. because, i found a webSite online which would produce the same feature for a Sequence, and the only way to produce the same result was using the Numpy.round() function.

Round() function generates some results which is really close, but not the same thing. So, to stay on the safe side, i used the numpy version instead of the other one.